modules/rgbd/include/opencv2/rgbd.hpp

```diff
@@ -56,13 +56,13 @@ namespace rgbd
    * a limit. For a float/double, we just check if it is a NaN
    * @param depth the depth to check for validity
    */
-  CV_EXPORTS
+  CV_EXPORTS_W
   inline bool
   isValidDepth(const float & depth)
   {
     return !cvIsNaN(depth);
   }
-  CV_EXPORTS
```

Comment:
Do we really need `isValidDepth*` functions to be exported? I think `nan` checks can be done in Python.